lib/sudoers/src/ast.rs

```diff
@@ -39,15 +39,25 @@ pub struct RunAs {
 }
 
 /// Commands in /etc/sudoers can have attributes attached to them, such as NOPASSWD, NOEXEC, ...
-#[derive(Debug, Clone, PartialEq, Eq)]
-pub enum Tag {
-    NoPasswd,
-    Timeout(i32),
+#[derive(Debug, Clone)]
+#[cfg_attr(test, derive(PartialEq, Eq))]
+pub struct Tag {
+    pub passwd: bool,
```

Comment:
nit: I would personally have the AST closely match the syntax, i.e. make this `nopasswd` instead of inverting the polarity.

(the `chdir` field does not bother me that much because there's no DSL logic applied on top of the AST. though, we should be consistent and name the fields after the actual syntax, e.g. `cwd`, OR after the grammar, e.g. `chdir` but not mix the two)

Reply:
This is a nit I wasn't very decided upon; the cwd is actually a good point (I'll change that). For the passwd/nopasswd: there are various of these flags that we may want to add in the future (mail/nomail, follow/nofollow), whatever choice we make here I think the polarity should be consistent for all flags, and that made me go for `passwd = false` over `nopasswd = true` (and it avoids a double negation).